src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,56 @@
+.watch {
+  @include circle($watch-size);
+  min-width: $watch-size;
+  border: 2px solid rgb(15, 38, 78);
+  display: flex;
+  margin: auto;
+  position: relative;
+  box-shadow: 40px 10px 10px rgba(30, 79, 164, 0.3);
+  background: repeating-radial-gradient(#fff, #01405c 200px);
+
+  &__minute-hand {
+    height: $height-minute-hand;
+    width: $width-minute-hand;
+    position: absolute;
+    top: top($height-minute-hand);
+    right: right($width-minute-hand);
+    border-radius: 16px;
+    background: radial-gradient(#fff, #172830);
+    animation: getRound 3600s  linear 1;
+    transform-origin: bottom center ;
+  }
+
+  &__second-hand {
+    height: $height-second-hand;
+    width: $width-second-hand;
+    position: absolute;
+    top: top($height-second-hand);
+    right: right($width-second-hand);
```

Comment:
we can just add `transform: translate(-50%;-50%)` and `top:50%; left:50%`

Reply:
what do you think?

Reply:
Ok, no problem))